### DISCUSSION TITLE
Modifications serveur -> rollback version 2.3.1

### DISCUSSION THREAD

**Discussion by beng-git** [danymat/INSAnonym-utils#93] [Q&A, Bug report & Support]
Bonjour,

Suite aux divers problèmes liés à la version 2.4 du serveur, nous avons rollback vers la version 2.3.1 qui implémente les métriques telles que je vous les ai données dans le fil https://github.com/danymat/INSAnonym-utils/discussions/88#discussion-3677395 

Ce rollback a été effectué en début d'après midi et le serveur est de nouveau opérationnel. Vos soumissions ont été conservées, mais cette version des métriques devraient être plus favorables à l'anonymisation.

La conséquence du rollback est que le calcul des métriques est un peu plus long (a priori 15-20 mins).

Cordialement,

BN

**Comment by beng-git** [comment]
Bonjour,

Suite à investigation, certaines métriques sur le serveur pénalisent en fait davantage les DEL que les métriques que je vous ai diffusées, de manière proportionnelle aux nombre de DEL dans la soumission.

Cordialement,

BN

**Comment by ceichler** [comment]
Les métriques serveurs et l'explication des différences sont disponibles sur un thread dédié:
https://github.com/danymat/INSAnonym-utils/discussions/98#discussion-3706942
